product-cli/cmd/kubeconfig/create.go

```diff
@@ -0,0 +1,44 @@
+package kubeconfig
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	hyperShiftKubeConfig "github.com/openshift/hypershift/cmd/kubeconfig"
+)
+
+type options struct {
+	namespace string
+	name      string
+}
+
+// NewCreateCommand returns a command which can render kubeconfigs for HostedCluster
+// resources.
+func NewCreateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "kubeconfig",
+		Short:        "Renders kubeconfigs for HostedCluster resources.",
+		Long:         hyperShiftKubeConfig.Description,
+		SilenceUsage: true,
+	}
+
+	opts := options{}
+
+	cmd.Flags().StringVar(&opts.namespace, "namespace", opts.namespace, "A HostedCluster namespace. Will default to 'clusters' if a --name is supplied")
+	cmd.Flags().StringVar(&opts.name, "name", opts.name, "A HostedCluster name")
+
+	cmd.RunE = func(cmd *cobra.Command, args []string) error {
+		if opts.name != "" && opts.namespace == "" {
```

Comment:
just initialize to "clusters" in line 27

Reply:
Should I do that for the regular version as well (cmd/kubeconfig/create.go)?

Reply:
I just went ahead and updated it as well.